src/views/components/MoreSettings.jsx

```diff
@@ -79,6 +95,14 @@ export const MoreSettings = () => {
           },
         }}
       >
+        <ContextMenuItem
+          icon={<FullScreenIcon />}
+          label={`${isFullScreen ? "Exit" : ""} Full Screen`}
```

Comment:
An extra space is added in the front in case of isFullScreen being false

Reply:
Making the correction.